components/builder-api/src/config.rs

```diff
@@ -57,7 +58,8 @@ impl Default for Config {
                  ui:          UiCfg::default(),
                  memcache:    MemcacheCfg::default(),
                  jobsrv:      JobsrvCfg::default(),
-                 datastore:   DataStoreCfg::default(), }
+                 datastore:   DataStoreCfg::default(),
+                 kafka:       KafkaCfg::default(), }
```

Comment:
FYI, since every field of the `Config` struct implements `Default`, you can just derive `Default` for `Config` and get rid of this entire implementation block.

Reply:
that's beautiful thanks!